janusgraph-hadoop-parent/janusgraph-hadoop-2/pom.xml

```diff
@@ -50,22 +58,32 @@
         <dependency>
             <groupId>org.apache.hbase</groupId>
             <artifactId>hbase-server</artifactId>
-            <version>${hbase098.version}</version>
+            <version>${hbase100.version}</version>
             <optional>true</optional>
             <scope>test</scope>
             <exclusions>
                 <exclusion>
                     <groupId>org.mortbay.jetty</groupId>
                     <artifactId>servlet-api-2.5</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.lmax</groupId>
+                    <artifactId>disruptor</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.hbase</groupId>
             <artifactId>hbase-client</artifactId>
-            <version>${hbase098.version}</version>
+            <version>${hbase100.version}</version>
             <optional>true</optional>
             <scope>test</scope>
+            <exclusions>
```

Comment:
com.lmax.disruptor is only in hbase-server.  No need to exclude for hbase-client.